ElectronNET.Host/ElectronHostHook/package.json

```diff
@@ -7,12 +7,15 @@
   },
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "tsc -p ."
   },
   "keywords": [],
   "author": "Gregor Biswanger",
   "license": "MIT",
   "devDependencies": {
+    "@microsoft/signalr": "^6.0.2",
+    "@types/signalr": "^2.2.37",
```

Comment:
@microsoft/signalr ships with its own type definitions. @types/signalr is for the older ASP.NET SignalR and is not compatible with @microsoft/signalr

Reply:
@BrennanConroy Thank you, thats correct, I've started a while ago with aspnet/signalr and just updated the packages - It's fixed :-)